### ISSUE TITLE
Night mode white background fickering

### ISSUE THREAD

**Issue by littlemuyer** [ankidroid/Anki-Android#4253]
Currently AnkiDroid (with night mode enabled) makes a white background flickering each time a card is changed to the next.
It's very irritating and defeats the purpose of a night mode option.

What steps will reproduce the problem?
Open normal study for any deck in night mode and go thought the cards.

What is the expected output? What do you see instead?
Black colored background always. A white flickering background each time I press next card or show answer.

Does it happen again every time you repeat the steps above? Or did it
happen only one time?
Yes. Happens always.

What version of AnkiDroid are you using?
2.6 alpha 30

I know from others' reports this issue has been reopen multiples times since version 2.0.1

**Comment by timrae** [comment]
I can't reproduce, could you please make a video? 
What version of Android? What device? From what version of AnkiDroid did it start happening?

**Comment by littlemuyer** [comment]
I forgot to mention that the I have:
Sony Xperia U ST25i (crappy, I know put doesn't have visible problems)
with 
Android version 4.0.4

how do I do the video?
But anyway, the video wouldn't help much.
I just open some deck, tap next card, next and next and in everyone there's a white flickering.

All that in night mode.

**Comment by timrae** [comment]
*From what version of AnkiDroid did it start happening?*

> how do I do the video?

You could just record the video with another camera if you one, and then upload to Google Drive.

> Sony Xperia U ST25i

Are you just using the standard ROM?

**Comment by littlemuyer** [comment]
I read in some old AnkiDroid google code page that it started in AnkiDroid 2.0.1.
But for me it has always ben like this ( some version above 2.0.1 )


For ROM you mean internal memory (no memory cards)?
Then yes, it uses the internal memory.

**Comment by timrae** [comment]
> For me it has always been like this

There used to be a bug where the screen flickered white upon first opening the Reviewer, which I fixed several versions ago (maybe 2.4?). The issue you describe doesn't exist on any of the devices we tested, or on the official Android emulator so I can only assume that it's an issue with your specific device, in which case there's not really anything we can do about it sorry.

Your device is very old, so if it's very important to you then I suggest you buy a new device. I am using a Nexus 5x on almost permanent night mode and it definitely doesn't happen here.

(Note: duplicate of #2293)
